index.html

```diff
@@ -81,148 +81,155 @@ <h4 class="list-heading">Skills</h4>
-                            <li><img src="images/github1.png" alt="github" class="icons"></li>
-                            <li><img src="images/linkden.png" alt="linkden1" class="icons "><img src="images/linkden2.png"
-                                    alt="linkden1" class="icons "></li>
-                            <li><img src="images/1.png" alt="1" class="icons"></li>
-                            <li><img src="images/twitter1.png" alt="twitter" class="icons"></li>
-                            <li><img src="images/2.png" alt="2" class="icons"></li>
-                        </ul>    
-                    </div>
-                </form>
-            </div>         
-    </div>
-    <div class="contact-email">
-        <ul>
-            <li><p>emialhere@email.com</p></li>
-        </ul>
-    </div>
-    
-</footer>
+            </div>
+        </section>
+        <footer>
+            <div class="template-footer">
+                <h1 class="mobile-us">CONTACT US</h1>
+                <h1 class="desktop-us">Let's Chat!</h1>
+                <p>If you have an application you are
+                    interested in developing, a feature that you
+                    need built or a project that needs coding. <br>
+                    I’d love to help with it </p>
+
+                <div class="form-section">
+                    <form class="form-submit" action="https://formspree.io/f/mjvzkrzj" method="post">
+
+                        <input type="text" id="name" name="user_name" placeholder="Name" maxlength="30" required>
+
+                        <input type="email" id="email" name="email" placeholder="Email" required value>
+
+                        <textarea name="Enter text here" id="msg" placeholder="text" required cols="30" rows="10"
+                            maxlength="500"></textarea>
+
+                        <div>
+                            <button class="form-button" type="submit" onclick>
+                                See Project
+                            </button>
+                        </div>
+                        <div class="footer-hr">
+                            <hr>
+                        </div>
+                        
+                    </form>
+                </div>
+            </div>
+            <div class="link-page">
+               
+                <ul class="icon">
+                    <li><img src="images/github1.png" alt="github" class="icons"></li>
+                    <li><img src="images/linkden.png" alt="linkden1" class="icons "><img
+                            src="images/linkden2.png" alt="linkden1" class="icons "></li>
+                    <li><img src="images/1.png" alt="1" class="icons"></li>
+                    <li><img src="images/twitter1.png" alt="twitter" class="icons"></li>
+                    <li><img src="images/2.png" alt="2" class="icons"></li>
+                    <li><a href="contact-mail">emialhere@email.com</a></li>
+                </ul>
```

Comment:
- [ ] Please  🙏🏽 review the source of the icons to show them in the browser.

![image](https://user-images.githubusercontent.com/58642949/187030119-1344eddb-2625-45d7-8329-edde46ff01c1.png)